kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue

```diff
@@ -1002,7 +1002,7 @@
       },
       allowLearnersDownloadDescription: {
         message:
-          "Allow users to explore resources they don't have and mark it for Kolibri to automatically download when it's available in their network.",
+          'Allow users to explore resources they don’t have and mark it for Kolibri to automatically download when it’s available in their network.',
```

Comment:
After discussing with @rtibbles, we propose to reword as:

`'Allow users to mark resources for Kolibri to download when available in their network.'`